social_core/tests/backends/test_saml.py

```diff
@@ -26,8 +29,8 @@
 DATA_DIR = path.join(path.dirname(__file__), 'data')
 
 
-@unittest2.skipIf('TRAVIS' in os.environ and sys.version_info[:2] == (3, 5),
-                  'Travis-ci segfaults in Python 3.5 probably due to a bad ' \
+@unittest2.skipIf('TRAVIS' in os.environ,
```

Comment:
I gave up. It segfaults only when run within tox and on travis machine.
It seems #13 is better after all.